### PR TITLE
Add getRenderTree() to Flutter Driver

### PR DESCRIPTION
**Pull request by cbracken**
Returns a dump of the application's render tree.

### CODE REVIEW

**Comment by cbracken** [comment]
Fixes #7006

File: packages/flutter_driver/lib/src/render_tree.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+class RenderTree extends Result {
+  /// Creates a [RenderTree] object with the given string representation.
+  RenderTree(this.tree);
+
+  /// Deserializes the result from JSON.
+  static RenderTree fromJson(Map<String, dynamic> json) {
+    return new RenderTree(json['tree']);
+  }
+
+  final String tree;
```
Comment (yjbanov) [inline packages/flutter_driver/lib/src/render_tree.dart]: `package:flutter_driver` has 100% public doc coverage, so let's document all new public members. Although it's not enforced yet.

**Review by yjbanov — APPROVED** [review]

File: packages/flutter_driver/lib/src/render_tree.dart (hunk 1 above)
Comment (cbracken) [inline packages/flutter_driver/lib/src/render_tree.dart]: Done.

**Pull request merged**